src/test/goldenText.ts

```diff
@@ -50,8 +50,20 @@ export class GoldenText {
       const match = frame.match(/^(.*):(\d+):(\d+)$/);
       if (!match) return null;
       const filePath = match[1];
-      if (filePath === __filename) continue;
-      return filePath;
+
+      // We need to consider .ts files because we may have source-map-support to convert stack traces to .ts
```

Comment:
This seems dangerous in tests, where precise behavior js-debug's of source maps handling is tested

Reply:
As discussed offline, this only affect golden test files